index.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>mint login form clone</title>
+</head>
+<body>
+    <main>
+        <div id="sign-in-btn-info">
+            <div id="text-info">
+                <div id="question">
+                    <p>Already have an Intuit account?</p>
+                </div>
+                <div id="user-id-password">
+                    <a href="#">
+                        <p> I forgot my User ID or Password</p>
+                    </a>
+                </div>
+            </div>
+            <button  id="sign-in-btn">
+                <i class="fa fa-lock"></i>
+                <span>Sign in</span>
+            </button>
+        </div>
+    <div class="header">
+        <div class="header-links">
+            <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26adobe_mc%3DMCMID%253D08497889761544046421391270368961118431%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1562778150%26ivid%3D538f0975-5631-4a09-93f2-0c8809828c71"><img src="img/intuit.png" alt="intuit"></a>
+
+            <div id="link-imgs">
+                <a href="https://www.mint.com/"><img src="img/mint.png" alt="mint"><span class="product-name">mint</span></a>
```

Comment:
`AWESOME`

Nice job, :clap: :star2:  adding an `alt` attribute to the `<img>` element, this is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)